torchvision/models/detection/roi_heads.py

```diff
@@ -461,11 +461,10 @@ def _onnx_paste_masks_in_image_loop(masks, boxes, im_h, im_w):
     return res_append
 
 
-def paste_masks_in_image(masks, boxes, img_shape, padding=1):
-    # type: (Tensor, Tensor, Tuple[int, int], int) -> Tensor
+def paste_masks_in_image(masks: Tensor, boxes: Tensor, img_shape: List[int], padding: int = 1) -> Tensor:
```

Comment:
I'm only typing this function for now, will follow up this file in new PR.
This needed as this raises error otherwise.
Again here the `img_shape` is `Tuple[int, int]` which cannot  be as `resize_boxes` and `resize_keypoints` `use List[int]`
We need consistency here.

Reply:
Since this file is currently ignored by `mypy`, let's not introduce annotations here.

Reply:
Well I have to annotate it due to the `img_shape` which should be Tuple[int, int], Otherwise mypy will compain in other places.